test/unit/gateways/redsys_rest_test.rb

```diff
@@ -0,0 +1,269 @@
+require 'test_helper'
+
+class RedsysRestTest < Test::Unit::TestCase
+  include CommStub
+
+  def setup
+    @credentials = {
+      login: '091952713',
```

Comment:
Could we use fixtures for these creds too?

Reply:
I'll move these to the fixture creds!

Reply:
Done!